fwoa-tools/src/export-script.py

```diff
@@ -61,210 +67,212 @@
-            return False
-    return True
-
-def is_internal_reference(reference, server_url):
-    if reference.startswith(server_url):
-        reference = removeprefix(reference, server_url)
-    reference_split = reference.split('/')
-    if len(reference_split) == 2:
-        return True
-    return False
-
-def deep_get(resource, path):
-    temp = [resource]
-    for p in path:
-        new_temp = []
-        for item in temp:
-            if item is None or p not in item:
-                continue
-            if isinstance(item[p], list):
-                new_temp.extend(item[p])
-            else:
-                new_temp.append(item[p])
-        temp = new_temp
-    return temp
-
-def is_included_in_group_export(resource, group_member_ids, group_patient_ids, compartment_search_params, server_url, transitive_reference_ids=set()):
-    # Check if resource is part of the group
-    if resource['id'] in group_member_ids or resource['id'] in transitive_reference_ids:
-        return True
-    # Check if resource is part of the patient compartment
-    if resource['resourceType'] in compartment_search_params:
-        # Get inclusion criteria paths for the resource
-        inclusion_paths = compartment_search_params[resource['resourceType']]
-        for path in inclusion_paths:
-            reference = deep_get(resource, path.split("."))
-            for ref in reference:
-                if is_internal_reference(ref['reference'], server_url) and ref['reference'].split('/')[-1] in group_patient_ids:
-                    return True
-    return False
-
-# Transitive reference are resources referenced in group members or patient compartment resources
-# Transitive reference is extracted based on file bulkExport/schema/transitiveReferenceParams.json
-# Update the file and redeploy to change transitive reference to be included
-def get_transitive_references(resource, transitive_reference_map, server_url):
-    if resource['resourceType'] in transitive_reference_map:
-        path_map = transitive_reference_map[resource['resourceType']]
-        generated_transitive_refs = []
-        for path, target_type in path_map.items():
-            targets = deep_get(resource, path.split('.'))
-            generated_transitive_refs.extend([target['reference'] for target in targets if is_internal_reference(target['reference'], server_url)])
-        resource['_generated_transitive_refs'] = generated_transitive_refs if len(generated_transitive_refs) !=0 else None
-    return resource
+    filtered_tenant_id_frame_with_composite_id = original_data_source_dyn_frame.filter(
+        f=lambda x: x['_tenantId'] == tenantId
+    )
 
-datetime_transaction_time = datetime.strptime(transaction_time, "%Y-%m-%dT%H:%M:%S.%fZ")
+    filtered_tenant_id_frame = filtered_tenant_id_frame_with_composite_id.map(
+        f=remove_composite_id)
 
-if (group_id is None):
```

Comment:
Is this intended? are we supposed to delete `group_id` code?

Reply:
Yes, since we're only doing system level exports for this tool, we got rid of all group export code since it was unnecessarily cluttering the code.